plugins/grpc/swift-protobuf/v1.0.0/buf.plugin.yaml

```diff
@@ -0,0 +1,23 @@
+version: v1
+name: buf.build/grpc/swift-protobuf
+plugin_version: v1.0.0
+source_url: https://github.com/grpc/grpc-swift-protobuf
```

Comment:
no `integration_guide`, currently: the one [linked to in the old repo](https://github.com/bufbuild/plugins/blob/c6e380cb6f6ff50a05d3101396d5e5e66dec51b7/plugins/grpc/swift/v1.24.2/buf.plugin.yaml#L5) didn't look valid anymore. We could maybe point over to the [Swift Package Index docs](https://swiftpackageindex.com/grpc/grpc-swift-protobuf/1.0.0/documentation/grpcprotobuf)?

Reply:
resolved in https://github.com/bufbuild/plugins/pull/1723/commits/71e437ea3297b90cde4cea6c73276b932a0d1835